server/service/apple_mdm.go

```diff
@@ -1603,10 +1603,19 @@ func (svc *Service) EnqueueMDMAppleCommandRemoveEnrollmentProfile(ctx context.Co
 		return ctxerr.Wrap(ctx, err, "getting host info for mdm apple remove profile command")
 	}
 
-	if h.Platform == "ios" || h.Platform == "ipados" {
+	switch h.Platform {
+	case "ios":
+		fallthrough
```

Comment:
Nit (doubly-so since it's merged!) but for the future, you can have multiple values in a case which is clearer/cleaner when the other case is made of only `fallthrough` IMO, e.g. https://go.dev/play/p/-uqeCAX3c7x